pkgs/applications/science/electronics/kicad/kicad-ngspice.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/eeschema/sim/ngspice.cpp b/eeschema/sim/ngspice.cpp
```

Comment:
Where did you get this patch? Is it possible to download it using `fetchpatch`?

Reply:
It comes from a comment in https://github.com/NixOS/nixpkgs/issues/63798
So I do not think it can be downloaded through fetchpatch.